components/Tooltip/Tooltip.jsx

```diff
@@ -43,17 +43,18 @@ const Tip = styled.div`
 
 const TipText = styled.span`
   display: inline-block;
-  max-width: 250px;
   overflow: hidden;
   text-overflow: ellipsis;
-  white-space: nowrap;
+  white-space: pre;
```

Comment:
This line has been added for the Tootip component accept `\n` as enter.

Reply:
what do you think to use `pre-wrap` instead?

Reply:
without `min-width` this change breaks the layout